content/en/agent/guide/how_remote_config_works.md

```diff
@@ -84,14 +78,14 @@ The following products and features are supported with Remote Config:
 
 ## Security Considerations
 
-Datadog implements the following safeguards, designed to protect the confidentiality, integrity, and availability of configurations received and applied to your Agents and tracing libraries:
+Datadog implements the following safeguards to protect the confidentiality, integrity, and availability of configurations received and applied by your Datadog components:
 
 * Agents deployed in your infrastructure request configurations from Datadog.
 * Datadog never sends configurations unless requested by Agents, and only sends configurations relevant to the requesting Agent.
 * Because the configuration requests are initiated from your Agents to Datadog over HTTPS (port 443), there is no need to open additional ports in your network firewall.
 * The communication between your Agents and Datadog is encrypted using HTTPS, and is authenticated and authorized using your Datadog API key.
-* Only users with the right RBAC permissions are authorized to enable Remote Configuration capability on the API key and use the supported product features.
-* Your configuration changes submitted through the Datadog UI are signed and validated on the Agent and tracing libraries, verifying integrity of the configuration.
+* Only users with the [`api_keys_write`][3] permissions are authorized to enable or disable Remote Configuration capability on the API key and use the supported product features.
```

Comment:
Above this line, can we also add below?

Only users with the [org_management](https://docs-staging.datadoghq.com/aliciascott/how_rc_works/account_management/rbac/permissions/#access-management)  permission are authorized to enable or disable Remote Configuration at the organization level.

Reply:
Is this something you want to highlight under a separate "Permissions" section?

Reply:
This was actually already there on the latest staging doc so I'll mark this as done.